Makefile

```diff
@@ -174,6 +174,12 @@ extension-release:
 		-DBUILD_KUZU=FALSE \
 	)
 
+shell-test:
+	$(call run-cmake-release, \
+		-DBUILD_SHELL=TRUE \
+	)
+	python3 -m pytest -v tools/shell/test
```

Comment:
I would switch to the tools/shell/test directory, and run pytest there directly.